onyo/lib/consts.py

```diff
@@ -0,0 +1,4 @@
+# TODO: pseudo-key generation mapping
+NEW_PSEUDO_KEYS = ['path']  # TODO: name temporary b/c RF (old idea of pseudo-keys doesn't match)
```

Comment:
I don't think there is a problem with this name.

IIRC, our decision was to no longer use the term "pseudo key" when referring to the keys in the filename. Since they, in fact, only contain content that is in keys.

However, we do intend "pseudo" keys that are queryable, etc. The `path` (aka filename) is among those (as well as some git related stuff, etc).

So I think `PSEUDO_KEYS` (without the `NEW_`) is an appropriate name for this const.

Reply:
Agree to this as well. The `NEW_` needs to change. That's why the comment reads "temporary name", because there is a constant `PSEUDO_KEYS` already that needs to go. Just a matter of the refactoring effort. It's still used in `get` (or its helper functions), so renaming is supposed to happen when the old stuff is gone entirely.

Reply:
Ahh; I did not check for that. The `NEW_` seemed like an odd "temporary" prefix.